### ISSUE TITLE
The issue about @slim.add_arg_scope

### ISSUE THREAD

**Issue by zzzzzz0407** [ailias/Focal-Loss-implement-on-Tensorflow#3]
I'm sorry that I am a newcomer to tensorflow, i am no familiar with the func@slim.add_arg_scope at about line 45, is that mean you want to share the default params to the function  (_inverted_residual_bottleneck), but if it just what i think, why do you not set the same op@slim.add_arg_scope just at the func(mobilenet_v2_base)? And why we should set the @slim.add_arg_scope?I just think that slim.conv2d and slim.separable_conv2d can get the default params automatically. May the issue is foolish, please help me!

**Comment by ChiefGodMan** [comment]
I'm sorry I don't understand your question. Can you give me more information about mobilenet_v2_base code ? 
Nevertheless, you don't need to change any other code except replacing your loss function with focal loss.
Best regards!
